### DISCUSSION TITLE
Email verification had localhost in the link instead of server domain

### DISCUSSION THREAD

**Discussion by usamashafiqs** [dani-garcia/vaultwarden#3236] [Q&A] [answered]
### Deployment environment

* vaultwarden version: 1.27.0

* Install method: Docker

* Clients used: Web
* Reverse proxy and version: Traefik v2.x

### Email verification had localhost in the link instead of server domain

![Screenshot 2023-02-09 015137](https://user-images.githubusercontent.com/18592683/217649178-79777340-a460-453d-95c1-4a7ca4299a68.png)

**Comment by BlackDex** [comment, marked as answer]
Configure the `DOMAIN` variable to the correct host.
